paper.tex

```diff
@@ -78,9 +78,9 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{INTRODUCTION}
 
-In the advent of connected vehicles and autonomous transportation, equipping consumer vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems can be prohibitively expensive.
-Technologies such as Radar, LIDAR and high-end cameras that are typically used for collision prediction are likely too expensive for most consumers.
-On the other hand, dash-cams, which are cheap video cameras that can be mounted to a car's dashboard, have become increasingly affordable and prevalent as a means to capture a driver’s perspective.
+In the advent of connected vehicles and autonomous transportation, we are equipping vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems.
```

Comment:
Change this to "In the advent of connected vehicles and autonomous transportation, it has become commonplace to equip vehicles with the necessary equipment to facilitate high-end automated collision avoidance systems."